src/pages/VideoCall/index.js

```diff
@@ -77,17 +85,28 @@ const VideoCallFrame = () => {
         })
 
         daily
+            .on('loading', roomLogsToSend)
+            .on('loaded', roomLogsToSend)
+            .on('started-camera', roomLogsToSend)
+            .on('camera-error', roomLogsToSend)
+            .on('active-speaker-change', roomLogsToSend)
+            .on('active-speaker-mode-change', roomLogsToSend)
+            .on('error', roomLogsToSend)
             .on('joined-meeting', eventActions)
             .on('participant-joined', eventActions)
             .on('participant-updated', eventActions)
+            .on('network-connection', roomLogsToSend)
+            .on('network-quality-change', roomLogsToSend)
             .on('participant-left', (event) => {
+                sendCallLogs(videoName, event)
                 setParticipantStatus(
                     !event.participant.local &&
                         Object.keys(daily.participants()).length < 2
                         ? t('participant-left')
                         : ''
                 )
             })
+            .on('left-meeting', roomLogsToSend)
 
         leaving.current.addEventListener('click', () => {
```

Comment:
for `leave` button you can use `onClick` event directly and you don't need to use ref

Ex
```
const leaveClick = () => daily.leave()

//button instead
<div className="control red leave" onClick={leaveClick}>
                            <img src={leave} alt="" />
                            <p>{t('leave')}</p>
                        </div>
```

Reply:
daily is only accessible inside useEffect though, so leaveClick would have to be defined inside useEffect right ? When I do so, leaveClick (as onClick value) is said to be "not defined"
 
![image](https://user-images.githubusercontent.com/50984435/80238661-1759d600-865f-11ea-8099-9ab587f99ace.png)

Reply:
Ok. My bad